src/osp/Active/opengl/SysRenderGL.cpp

```diff
@@ -27,6 +27,10 @@
 
 #include "../../Shaders/FullscreenTriShader.h"
 
+#include "../../Resource/resources.h"
```

Comment:
Food for thought for a later change.

Instead of using relative paths like this, we should have a folder for header files so that they can be included like

<osp/component/header.h>

which would allow, in the future, for the code of osp-magnum to be further decomposed into libraries that can be compiled against package-manager installed headers from the OS, without needing to do any extra work.

Reply:
This is already used in the other files that aren't in src/osp, they should work here too.